tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -185,7 +185,10 @@ def create_parser(product_choices=None):
                                  help="Path or url to symbols file used to analyse crash minidumps.")
     debugging_group.add_argument("--stackwalk-binary", action="store", type=abs_path,
                                  help="Path to stackwalker program used to analyse minidumps.")
-
+    debugging_group.add_argument("--output-directory", action="store",
```

Comment:
This argument is **extremely** Chromium-specific, and it seems odd to add it to WPT. If we really need it, we should add it to `Chrome-specific` rather than `Debugging` I think, but I wonder if we could figure out a way to avoid that? Perhaps adding a `stackparser-script-arg` similar to `webdriver-arg` or `binary-arg` ?

Reply:
How about I reuse the "--symbols-path" argument? The output directory is actually used to get the symbols.

Reply:
That sounds reasonable to me, though I'd like @jgraham 's input. The concepts are very similar (I believe tombstones just are what Android call minidumps/crash dumps, right?), to the point where I would be tempted to merge `--stackwalk-binary` and `--stackparser-script` - except that deprecating flags is hard :(

Reply:
Stephen, the parser we are using is a python script, so maybe call it "--stackwork-binary" is not correct?

Reply:
I think `--stackwalk-binary` has been a Python script for Firefox too. I'd definitely merge `--stackparser-script` with `--stackwalk-binary` since they're apparently both programs that get called to ingest stack dumps.

I also think that `--output-directory` is an overly general name for something quite specific. It looks like for Firefox we figure out the dump directory relative to the profile path [1]. The symbols path holds the data required for symbolification of the stack traces. So we can reuse that if this is actually the directory with the symbols rather than the directory to which dump files are written.

[1] https://searchfox.org/mozilla-central/source/testing/web-platform/tests/tools/wptrunner/wptrunner/browsers/firefox.py#800-812

Reply:
Thanks for the information. I will have a separate patch to update this as this one is already merged.